Description: Set debian paths
Forwarded: no
Author: Romain Beauxis <toots@rastageeks.org>
Last-Update: 2016-02-13
--- a/Makefile
+++ b/Makefile
@@ -12,8 +12,8 @@ WIN32_CFLAGS = -g -Wall -DWIN32 -I"c:\Pr
 WIN32_LDOPTS = -lwpcap -lwsock32 -L"c:\Program Files\WpdPack\Lib"
 WIN32_PT_OBJS = ptunnel.obj md5.obj
 
-prefix		= /usr
-bindir		= $(prefix)/bin
+prefix		= $(DESTDIR)/usr
+bindir		= $(prefix)/sbin
 mandir		= $(prefix)/share/man/man8
 
 all: ptunnel
